core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java

```diff
@@ -104,7 +124,7 @@ public void importData() throws Exception
         FileReader reader =  new FileReader(clinicalDataFile);
         BufferedReader buff = new BufferedReader(reader);
         List<ClinicalAttribute> columnAttrs = grabAttrs(buff);
-
+        
```

Comment:
I think there should be an `if` here that does this check based on the value of `this.attributesDatatype`. E.g.
```java
int patientIdIndex = -1;
int sampleIdIndex = -1;
if (this.attributesDatatype.equals("SAMPLE_ATTRIBUTES") //use a constant here
  sampleIdIndex = findSampleIdColumn(columnAttrs);
else if (this.attributesDatatype.equals("PATIENT_ATTRIBUTES") 
  patientIdIndex = findPatientIdColumn(columnAttrs);
...
```